src/euroeval/templates/constants.py

```diff
@@ -0,0 +1,177 @@
+"""Constants used for the template configurations."""
+
+NER_TAG_MAPPING = {
```

Comment:
I would probably move these label mappings into their respective template modules, to avoid spreading all the localisations out too much.